polar_area_chart_plugin/src/dp_polar_area_chart_plugin/components/PolarAreaChart/PolarAreaChartWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+// (C) 2023 GoodData Corporation
+
+import React, { useMemo } from "react";
+import { useDashboardSelector, IDashboardInsightProps, selectIsInViewMode } from "@gooddata/sdk-ui-dashboard";
+
+import { useHideOriginalWidgetTitleInViewMode } from "../useHideOriginalWidgetTitleInViewMode.js";
+import { CustomTitle } from "./CustomTitle.js";
+import { WIDGET_TITLE_SUFFIX } from "../../Plugin.js";
+import ErrorBoundary from "../ErrorBoundary/ErrorBoundary.js";
+import { PolarAreaChart } from "./PolarAreaChart.js";
+
+import "./styles.css";
+
+const PolarAreaChartWrapper: React.FC<IDashboardInsightProps> = (props) => {
+    const { LoadingComponent, ErrorComponent, widget } = props;
+    const isInViewMode = useDashboardSelector(selectIsInViewMode);
+
+    const GeneralError = useMemo(
+        () => <ErrorComponent message="Unable to display the report" />,
+        [ErrorComponent],
+    );
+
+    // replace original widget title with a new one without the suffix
+    useHideOriginalWidgetTitleInViewMode(widget, isInViewMode);
+    const customWidgetTitleComponent = useMemo(() => {
```

Comment:
Just wondering why something like this would not work here instead of another memo? 🤔 is there a specific reason? 

`return (
        <ErrorBoundary ErrorComponent={GeneralError}>
            {isInViewMode &&  <CustomTitle title={widget.title.replace(WIDGET_TITLE_SUFFIX, "")}/> }
            <div className={"polar-area-wrapper"}>
            ...`